### DISCUSSION TITLE
Enterprise repository 404

### DISCUSSION THREAD

**Discussion by NiklasBr** [pimcore/pimcore#16712] [Q&A / Support] [answered]
Trying to install from the enterprise server this morning gives the following error:

Failed downloading pimcore/pimcore, trying the next URL (404: The "https://enterprise.repo.pimcore.com/dists/pimcore/pimcore/11.1.5.1/f2149f6db9618d5afcccb546f05584f84189c8fb.zip" file could not be downloaded (HTTP/2 404 ))

Is this a known issue?

**Comment by ghost** [comment]
@NiklasBr Thanks for reporting. Yes, we are aware of this and already working on a solution.
If possible, please report such issues via the enterprise portal where we can help you more detailed. In case you need access to it, please contact your Pimcore partner manager. Thanks a lot!

**Reply by NiklasBr** [reply]
I much prefer to work in the open because it is more efficient and reduces duplicate and triple work for everyone involved. Feel free to send status updates preventively in case of issues like this.

**Reply by NiklasBr** [reply]
@wisconaut we have not yet been given access to the enterprise portal and the enterprise repo is still reporting a 404 error, is there an ETA or workaround you can share?

**Reply by fashxp** [reply]
Please contact your Partner Manager to get access. 
We simply cannot scan the whole discussions forum for eventually urgent enterprise support topics. Thank you for your understanding. 

Repman (service for the enterprise repository you are using) is up again and working.

**Reply by NiklasBr** [reply]
Our partner manager contacted you yesterday (Friday midday) and you have not yet responded.

**Reply by NiklasBr** [reply, marked as answer]
Today I was given access to Enterprise Edition Support.

I will file issues there primarily in the future, but I stand by my comment that it is more efficient, less work for everyone involved, and matter of confidence if you allow status issues to be discussed in locations such as this.

**Reply by ghost** [reply]
@NiklasBr Thanks a lot for the update. Of course we do not want you to stop discussing things here. We just wanted to point out that from an enterprise support perspective, we cannot scan all topics and discussions here and react accordingly, like fashxp wrote above. That's why we really ask you to raise enterprise issues in the enterprise support portal.

**Reply by NiklasBr** [reply]
The point I am trying to make is that even when logged into the enterprise portal I cannot see if zero or twenty other partners/customers have reported any issues such as service disruptions, which is why open forums or dashboards are an absolute necessity. If I report there, am I the twelfth one to do so the past hour or the first one? No way to prevent lots of duplicate work. If instead of locking everything down you offered a public insight into things like that it would be faster and more efficient for everybody.

**Comment by AndrewKMoses** [comment]
@fashxp We have tried in different ways to get help via Partner Manager, but so far we have no response / solution.

We are still having issues with this. In our case it is for Pimcore v6.9.7, for two different projects. It's as if the package no longer exists?

Package operations: 2 installs, 0 updates, 0 removals
  - Downloading pimcore/pimcore (v6.9.7)
 0/1 [>---------------------------]   0%    Failed downloading pimcore/pimcore, trying the next URL (404: The "https://lts.repo.pimcore.com/dists/pimcore/pimcore/6.9.7.0/21e447cddbce8a9e4d149295d93cf6e6cfaa4997.zip" file could not be downloaded (HTTP/2 404 ))
  - Downloading pimcore/pimcore (v6.9.7)
    Failed to download pimcore/pimcore from dist: The "https://api.github.com/repos/pimcore/pimcore-lts/zipball/21e447cddbce8a9e4d149295d93cf6e6cfaa4997" file could not be downloaded (HTTP/2 404 ):
{"message":"Not Found","documentation_url":"https://docs.github.com/rest/repos/contents#download-a-repository-archive-zip"}                                                                                                                                                                                         
    Now trying to download from source
  - Syncing pimcore/pimcore (v6.9.7) into cache
 1/1 [============================] 100%
                                                                                                                                                                                       
  [RuntimeException]                                                                                                                                                                   
  Failed to execute git clone --mirror -- 'https://31c/...305:x-oauth-basic@github.com/pimcore/pimcore-lts.git' '/var/www/.composer/cache/vcs/git-github.com-pimcore-pimcore-lts.git/'  
                                                                                                                                                                                       
  Cloning into bare repository '/var/www/.composer/cache/vcs/git-github.com-pimcore-pimcore-lts.git'...                                                                                
  remote: Repository not found.                                                                                                                                                        
  fatal: repository 'https://github.com/pimcore/pimcore-lts.git/' not found

**Reply by fashxp** [reply]
Your repository is outdated. 
Please contact me via email `firstname.lastname<at>pimcore.com` and share your company name as well as customer name. I'll forward accordingly.

**Reply by NiklasBr** [reply]
Why are things like this never communicated in advance? But instead have to be first discovered, and then debugged, and then waited on… For a response which could have been a quick email which was never sent? Nobody was told about these issues and changes, so this is _yet another_ example why the current state of Pimcore Enterprise requires better documentation and an open forum.

**Reply by fashxp** [reply]
Guys, our CAG Ateles dedicated Partner Manager was at your premises on Monday this week, the 11th; meeting with management; hence there was no information conveyed about the case you are mentioning. We are tracing if there was an error on our end; our executives are meeting next week at Inspire - are you joining too?
